web/src/views/Dashboards/Dashboards.vue

```diff
@@ -276,19 +276,17 @@ export default defineComponent({
       });
     });
 
-    const removeDashboard = async () => {
+    const deleteDashboard = async () => {
```

Comment:
Rust code, HTML labels, and (soon) the Open API specification use verb ‘delete’. There is no reason to deviate here.